code/style.css

```diff
@@ -35,24 +70,68 @@ p {
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  margin: 10px;
 }
 
 .room-deck {
   width: 250px;
   height: 400px;
   background: midnightblue;
-
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  margin: 10px;
+}
+
+h1 {
+  text-align: center;
+  font-size: 24px;
+  margin: 16px;
 }
 
 button {
   padding: 8px 16px;
   background: crimson;
   border: none;
   font-size: 20px;
+  color: white;
   border-radius: 8px;
   display: block;
-  margin: 0 auto;
+  margin: 16px auto;
 }
+
+footer {
+  background: black;
+  height: 80px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color:  white;
+  padding: 0 16px;
+  border-top: 1px solid;
+}
+
+footer div a {
+  text-decoration: none;
+  color: white;
+  margin-left: 16px;
+}
+
+
+/* Desktop */
+@media (min-width: 900px) {
+  .cards {
+    margin: 50px auto;
+    height: 600px; 
```

Comment:
This where you can fix this
```
Tried to make the styling mobile first. It got weird between 900-950px, the button sits in top of the Room-card there and I couldn't spend more time fixing it. Please let me know how to solve it?
```
So if you change it to `auto` it will be solved. The reason by it is that now you have said the height should always be 600px. When the width is 900px on the page the card will go to two rows - and they don't actually fit in 600px so that's why everyhing is on top of each other. With `auto` you give it the rooms that they want